lib/util.js

```diff
@@ -33,10 +33,17 @@ const downloadPmd = async function(version, token) {
 
 const executePmd = async function(pmdInfo, fileListOrSourcePath, ruleset, reportFormat, reportFile) {
     let pmdExecutable = '/bin/run.sh pmd';
+    if (isPmd7Cli(pmdInfo.version)) {
+        pmdExecutable = '/bin/pmd.sh';
```

Comment:
```suggestion
        pmdExecutable = '/bin/pmd';
```

Just checked - we don't use a extension...